### ISSUE TITLE
Dates on blog posts / version announcements

### ISSUE THREAD

**Issue by Apanatshka** [elm-lang/elm-lang.org#274]
Elm is on the front page of HN again with the Virtual Dom in Elm post again :smiley: I just noticed [this comment](https://news.ycombinator.com/item?id=9542114):

> Great post, as usual. I wish these posts had dates on them.

Maybe a good idea for the new website to have the date of a blog post on the blog post page. 

---------------------------------

Release  | Date        | Note
---------|-------------|---------------
0.15     | 2015-04-20  |
(0.14.1  | 2015-01-09  | Not a Blog post)
0.14     | 2014-12-10  |
0.13     | 2014-09-25  |
0.12.3   | 2014-05-20  |
0.12.1   | 2014-05-01  |
0.12     | 2014-03-24  |
0.11     | 2014-01-21  |
0.10.1   | 2013-12-30  |
0.10     | 2013-10-24  |
0.9      | 2013-08-13  |
0.8      | 2013-05-29  |
0.7.1    | 2013-01-25  |
0.7      | 2013-01-04  |
0.6      | 2012-12-08  |
0.5      | 2012-10-25  |
0.4      | 2012-09-18 |
0.3.6    | 2012-08-16 |
0.3.5    | 2012-06-30  |
0.3      | 2012-06-21  |

Article | Date | Note
--------|------|----------------------------------------
 Introducing Elm Reactor | 2014-09-24 |
 Blazing Fast HTML | 2014-07-23 |
 Elm’s Time Traveling Debugger | 2014-04-15|
 Elm package manager | 2014-01-01
 Elm REPL | 2013-09-03 |
 Hot-swapping in Elm | 2013-09-05 |
(Concepts behind the Elm Logo | | Not a blog post)
(Elm in VentureBeat | 2013-07-26 | Not a blog post on elm-lang.org)
 Elm ♥ Prezi | 2013-05-21? |
 Escape from Callback Hell | 2012-11-02 |
 Making Pong | 2012-12-14? |

**Comment by evancz** [comment]
I wonder if it's the same guy who always asks for this? In any case, can someone look through the history of this repo to figure it out? Ideally with a link to the history of changes for each blog post so we can see when they started / stabilized for release. @Apanatshka, can you take care of asking on the list if you can't do it?

**Comment by Apanatshka** [comment]
Oh I've never seen that comment before, but I figured it's a good thing to have. I'll ask on the mailinglist, see if we can do a group effort thing where whereone picks one or a few posts and writes it in comments here.

**Comment by Apanatshka** [comment]
I'm adding dates whenever HN and the Wayback Machine agree

**Comment by Apanatshka** [comment]
@evancz I'm think I'm done. I can't find a reliable source for release dates of some, those aren't checked and have a ? after the date I did find (e.g. on archive.org which can be inaccurate). I'll turn the lists into tables for easier use.

**Comment by evancz** [comment]
Awesome work, thank you so much! I am going to be doing a bit more work on the website redo, and part of what it means to be a blog post will be the date, so this is great :)

**Comment by evancz** [comment]
@Apanatshka, did you find the blog posts for 0.3, 0.3.5, and 0.3.6 online? I have actually lost the source for those, so it could be cool to reconstruct them. Same for all the content on testblogpleaseignore actually!

**Comment by Apanatshka** [comment]
The WayBack Machine was not very helpful for the oldest versions. I eventually found the links: http://www.testblogpleaseignore.com/2012/06/19/announcing-elm-0-3-modules/, http://www.testblogpleaseignore.com/2012/06/29/announcing-elm-0-3-5-javascript-integration-signal-filters-and-more/, http://www.testblogpleaseignore.com/2012/08/16/elm-0-3-6json-support-and-better-error-messages/. BTW: Just now I found the exact dates of version announcements I wasn't sure of among these links and another reddit thread, so that list is complete now. 
Everything from testblogpleaseignore is lost I think, because archive.org adhered to the robots.txt file and that blog had a restrictive one. 
BUT! I think you [copied blog post of the 0.3 announcement on the mailing list](https://groups.google.com/d/msg/elm-discuss/qXu-Y9TOTGE/V0ZeOQFbaKMJ). At least that what it looks like, with the background colour and all. So that's at least one.
